sklearn/linear_model/base.py

```diff
@@ -360,6 +425,13 @@ class LinearRegression(LinearModel, RegressorMixin):
 
     normalize : boolean, optional, default False
         If True, the regressors X will be normalized before regression.
+        When the regressors are normalized, the fitted `coef_` are the same
```

Comment:
I don't quite understand this. Did you mean the "fitted coef_s are of the same scale"?